scaffold/github/workflows/TestZizmor.yml

```
@@ -0,0 +1,28 @@
+name: Security Analysis with Zizmor
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["*"]
+
+jobs:
+  zizmor:
+    name: Run Zizmor Security Analysis
+    runs-on: ubuntu-latest
```

Pin the ubuntu version, so 26 doesn't break things later?

```suggestion
    runs-on: ubuntu-24.04
```